doc/source/development/extending.rst

```diff
@@ -95,7 +95,7 @@ on :ref:`ecosystem.extensions`.
 
 The interface consists of two classes.
 
-:class:`~pandas.api.extensions.ExtensionDtype`
+Class:~pandas.API.extensions.ExtensionDtype
```

Comment:
`:class:` is something that we use for sphinx, I don't think that this is something we want